src/KM_Points.pas

```diff
@@ -113,6 +113,7 @@   TKMTriMesh = record
 
 const
   ZERO_POINT: TKMPoint = (X: 0; Y: 0);
+  ZERO_POINT_F: TKMPointF = (X: 0.0; Y: 0.0);
```

Comment:
KMPOINT_INVALID is a bit confising, because point (-1, -1) is not invalid (we can use it as some adjustment f.e.). But its invalid point on Map. That is why I initially name it INVALID_MAP_POINT.

So may be KMPOINT_INVALID_ON_MAP or smth ?